Source/WebKit/UIProcess/API/haiku/PageClientImplHaiku.cpp

```diff
@@ -23,317 +23,320 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 #include "config.h"
-
 #include "PageClientImplHaiku.h"
 
-#if USE(COORDINATED_GRAPHICS)
-#include "DrawingAreaProxyCoordinatedGraphics.h"
-#endif
-
 #include "DrawingAreaProxy.h"
 #include "WebProcessProxy.h"
 #include "WebViewBase.h"
+#include <WebCore/Region.h>
```

Comment:
includes from WebKit should use quotes to avoid confusion with Haiku system ones which use <>. Some includes have identical names so it's important to follow this rule.